package.json

```diff
@@ -20,19 +22,21 @@
     "debug": "^4.3.1",
     "puppeteer": "^10.2.0",
     "ts-node": "^9.1.1",
-    "tslib": "^2.2.0",
+    "uuid": "^8.3.2",
```

Comment:
I think we might be able to get by without this dependency by leveraging `crypto.randomUUID()` in the page context. See my comment below.